Simplified/NYPLMigrationManager.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+
+/**
+Manages data migrations as they are needed throughout the app's life
+
+App version is cached in UserDefaults and last cached value is checked against current build version
+and updates are applied as required
+*/ 
+class MigrationManager: NSObject {
+  @objc static func migrate() {
+    // Fetch target version
+    let targetVersion = Bundle.main.infoDictionary!["CFBundleShortVersionString"] as! String
+
+    // Fetch and parse app version
+    let appVersion = NYPLSettings.shared().appVersion ?? ""
+    let appVersionTokens = appVersion.filter({ $0.isNumber || $0 == "." }).split(separator: ".").map({ Int($0)! })
+    
+    // Run through migration stages
+    if versionComparator(appVersionTokens, [3, 2, 0]) { // v3.2.0
+      migrate1();
+    }
+
+    // Migrate Network Queue DB
+    NetworkQueue.sharedInstance.migrate()
+
+    // Update app version
+    NYPLSettings.shared().appVersion = targetVersion
+  }
+
+  // Less-than comparator operation
+  private static func versionComparator(_ a: [Int], _ b:[Int]) -> Bool {
```

Comment:
It might be clearer to call the function something that implies the comment, so it it's clear above what it will return. Maybe `versionIsLessThan`?